### DISCUSSION TITLE
GTK vs Flutter vs WebKit Frameworks

### DISCUSSION THREAD

**Discussion by D0bhareach** [gtk-rs/gtk4-rs#1548] [Ideas]
I think gtk-rs require better support for developers.
Flutter is working on top of GTK ( I guess transpiring ), WebKit is part of GTK. And yet, there is a tonne of materials for Flutter: tutorials, YouTube,  docs and very scarce resources to get information about gtk-rs. As to WebKit Frameworks they are very intuitive and in general are easier to pick up on them ( or is it? ).
GTK is well established and reliable GUI Framework and strangely, especially for these who have no prior experience with GTK before,  it's the most mysterious of all. Is it OK and shall it stay this way?

**Comment by bilelmoussaoui** [comment]
Your question has a simple answer without going into details. Flutter gets a ton of money from Google where GTK doesn't at all.

**Reply by D0bhareach** [reply]
gtk-rs is Rust's Port to GTK Development. Are we GUI yet? Yes. I would like to use my chance and to say thank you and all who support this repository for your hard work.
